### PR TITLE
Added error handler middleware to the end of src/index.ts.

### PR DESCRIPTION
**Pull request by thaigillespie**

### CODE REVIEW

File: src/middlewares/errorHandler/errorHandler.middleware.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { Request, Response, NextFunction, ErrorRequestHandler } from 'express';
+
+const ErrorHandler: ErrorRequestHandler = (
+  err: Error,
```
Comment (AneeshJalan) [inline src/middlewares/errorHandler/errorHandler.middleware.ts:4]: Is Error a standard typescript type or should that also be imported from 'express'?

Comment (thaigillespie) [reply src/middlewares/errorHandler/errorHandler.middleware.ts:4]: I think it's a Typescript standard type.

File: src/middlewares/errorHandler/errorHandler.middleware.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  req: Request,
+  res: Response,
+  next: NextFunction
+) => {
+  res.status(500).send({ err, testing: 'testing!' });
```
Comment (godwinpang) [inline src/middlewares/errorHandler/errorHandler.middleware.ts:9]: take out the testing lol

Comment (thaigillespie) [reply src/middlewares/errorHandler/errorHandler.middleware.ts:9]: lol I didn't even notice, thanks for pointing it out.

**Pull request merged**